lib/builder-browserify.js

```diff
@@ -17,7 +20,7 @@ function configure (bundler, cfg) {
     }
 
     function register (type, o) {
```

Comment:
I can't believe I allowed a single letter var name to be merged sometime in the past. I really hope I didn't write that.